applications/serial_lte_modem/README.rst

```diff
@@ -4,6 +4,7 @@ nRF9160: Serial LTE modem
 #########################
 
 The Serial LTE Modem (SLM) application can be used to emulate a stand-alone LTE modem on the nRF9160.
+The application accepts both the modem-specific AT commands documented in the `nRF91 AT Commands Reference Guide <AT Commands Reference Guide_>`_ and proprietary AT commands that run on the application processor.
```

Comment:
```suggestion
The application accepts both the modem-specific AT commands documented in the `nRF91 AT Commands Reference Guide <AT Commands Reference Guide_>`_ and proprietary AT commands that is documented in "https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/serial_lte_modem/doc/AT_commands_intro.html" (nice link text)
```
Another suggestion is that we change the title (from the linked item) "AT commands" to "SLM AT commands" to easier differentiate between modem specific and SLM ?

Reply:
Updated.